modules/core/upgrades/v5/upgrades.go

```diff
@@ -0,0 +1,44 @@
+package v5
+
+import (
+	"strings"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	clientkeeper "github.com/cosmos/ibc-go/v3/modules/core/02-client/keeper"
+	clienttypes "github.com/cosmos/ibc-go/v3/modules/core/02-client/types"
+	host "github.com/cosmos/ibc-go/v3/modules/core/24-host"
+	"github.com/cosmos/ibc-go/v3/modules/core/exported"
+)
+
+// Localhost is the client type for a localhost client. It is also used as the clientID
+// for the localhost client.
+const Localhost string = "09-localhost"
+
+// UpgradeToV5 prunes the 09-Localhost client and associated consensus states from the ibc store
+func UpgradeToV5(ctx sdk.Context, clientKeeper clientkeeper.Keeper) {
+	clientStore := clientKeeper.ClientStore(ctx, Localhost)
```

Comment:
I wonder if there is a way to wipe the entire client store? I think you could iterate over just the client prefixed key, but I kinda like explicitly deleting consensus/client states